packages/app/src/specs/CreateSpecModal.vue

```diff
@@ -99,7 +100,9 @@ const helpLink = computed(() => {
 const specFileName = computed(() => {
   const extension = props.gql.currentProject?.fileExtensionToUse ?? 'js'
 
-  return getPathForPlatform(`cypress/e2e/filename.cy.${extension}`)
+  const fileName = props.gql.currentProject?.defaultSpecFileName ?? `cypress/e2e/filename.cy.${extension}`
+
```

Comment:
Would it be better to provide a warning message instead of scaffolding a new spec that didn't match the project's specPattern if `props.gql.currentProject?.defaultSpecFileName` is undefined?

Reply:
The warning message can be the someone as this one https://github.com/cypress-io/cypress/pull/20654#discussion_r832633978 where the spec name to create do not match - what do you think?

Reply:
+1 I think a warning message like you linked would better-align with what users would expect.